cpp/oneapi/dal/util/load_graph.cpp

```diff
@@ -21,8 +21,8 @@
 namespace oneapi::dal::preview {
 namespace detail {
 template <>
-edge_list<std::int64_t> load(const std::string &name) {
-    using int_t = std::int64_t;
+edge_list<std::int32_t> load(const std::string &name) {
```

Comment:
my general concern is on the place of the functionality, it is put under util folder which contradicts the definition of the dal as a lib of blocks for optimization of all stages of the pipeline, from data load till training, https://spec.oneapi.com/versions/latest/elements/oneDAL/source/introduction.html
Data source, numeric tables etc are introduced to optimize pipeline. The same applies to the graph. Graph loading is important part of pipeline and should be in the different place.
All such decisions should be part of design and then to be implemented in the code